test/functional/rpc_fundrawtransaction.py

```diff
@@ -509,13 +519,13 @@ def run_test(self):
         #now we need to unlock
         self.nodes[1].walletpassphrase("test", 600)
         signedTx = self.nodes[1].signrawtransactionwithwallet(fundedTx['hex'])
-        txId = self.nodes[1].sendrawtransaction(signedTx['hex'])
+        self.nodes[1].sendrawtransaction(signedTx['hex'])
         self.nodes[1].generate(1)
         self.sync_all()
 
         # make sure funds are received at node1
         assert_equal(oldBalance+Decimal('511.0000000'), self.nodes[0].getbalance())
-
+def test_many_inputs_fee(self):
```

Comment:
16293: missing whitespaces here and in other `def`s  below e.g.
```suggestion
    def test_many_inputs_fee(self):
```